### PR TITLE
product update and docs for apollo-router custom build versioning

### PR DESCRIPTION
**Pull request by dotansimha**
Closes https://github.com/graphql-hive/console/issues/3314 

## TODO 

- [x] fix build https://github.com/graphql-hive/console/pull/5940 
- [x] validate actual release from `main` (https://github.com/graphql-hive/console/pull/5938)
- [x] product update
- [x] changeset to cut v1 release of the hive plugin
- [x] update docs

### CODE REVIEW

File: packages/web/docs/public/apollo-router-download.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ parse_args() {
     key="$1"
 
     case $key in
-      -c | --commit)
+      -c | --commit | -v | --version)
```
Comment (dotansimha) [inline packages/web/docs/public/apollo-router-download.sh:14]: @kamilkisiela can you confirm it's going to work? 😆

**Review by saihaj — APPROVED** [review]

File: packages/web/docs/public/apollo-router-download.sh (hunk 1 above)
Comment (kamilkisiela) [reply packages/web/docs/public/apollo-router-download.sh:14]: you can use previews to verify it

File: packages/web/docs/src/pages/product-updates/2024-11-19-apollo-router-hive-versioning.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+(`x86_64`):
+
+```bash
+curl -fsSL https://graphql-hive.com/apollo-router-download.sh | bash
+```
+
+You may also specify a version using the `-v` or `--version` flag:
+
+```bash
+curl -fsSL https://graphql-hive.com/apollo-router-download.sh | bash -s -- --version router1.57.1-plugin1.0.0
```
Comment (kamilkisiela) [inline packages/web/docs/src/pages/product-updates/2024-11-19-apollo-router-hive-versioning.mdx:56]: Can you explain to the user where to find the list of versions?

File: packages/web/docs/src/pages/product-updates/2024-11-19-apollo-router-hive-versioning.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
```
[the remark this answers is not in the document]
Comment (dotansimha) [reply packages/web/docs/src/pages/product-updates/2024-11-19-apollo-router-hive-versioning.mdx:1]: done

**Pull request merged**